tests/driver_ir_nec/Makefile

```
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
+
+# required modules
+USEMODULE += ir_nec
+
```

```suggestion

#setup pin for ir-receiver
#this is the pin marked A0 on nucleo-f767zi
IR_PIN ?= GPIO_PIN(PORT_A,3)
CFLAGS += -D'IR_NEC_PARAM_PIN=$(IR_PIN)'

```
feel free to replace for the information why PORT_A, 4